### PR TITLE
[11.0][MIG] account_fiscal_position_vat_check: Migration to 11.0

### PR DESCRIPTION
**Pull request by rven**

### CODE REVIEW

**Comment by rven** [comment]
Tests are failing on Module account_lock_date_update: 0 failures, 1 errors

> odoo.exceptions.AccessError: ('The requested operation cannot be completed due to security restrictions. Please contact your system administrator.\n\n(Document type: account.update.lock_date, Operation: read)', None)

Which has nothing to do with my changes.

**Comment by pedrobaeza** [comment]
Yeah, indeed that's an error on current repository, so don't worry for now about that.

File: account_fiscal_position_vat_check/views/account_fiscal_position.xml [hunk 1]
```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+ © 2013-2016 Akretion (Alexis de Lattre <alexis.delattre@akretion.com>)
```
Comment (Tardo) [inline account_fiscal_position_vat_check/views/account_fiscal_position.xml:3]:
```suggestion
 Copyright 2013-2016 Akretion (Alexis de Lattre <alexis.delattre@akretion.com>)
```

**Review by Tardo — APPROVED** [review]
LGTM, only a little change...

**Comment by pedrobaeza** [comment]
Well, let's move on for having this module migrated.

**Pull request merged**